example/example.js

```diff
@@ -0,0 +1,23 @@
+const { AudioProcessor } = require('../index'); // Assuming your index.js is in the parent directory
+
+async function runExample() {
+  const audioFilePath = '../audio/a.mp3';
+  const outputFilePath = './redacted_audio.wav';
+
+  try {
+    // Create a transcriber and a redactor from the factory
+    const transcriber = AudioProcessor.getTranscriber('deepgram',process.env.DEEPGRAM_API_KEY, audioFilePath);
+    const redactor = AudioProcessor.getRedactor('deepgram', audioFilePath);
+
+    // Perform transcription
+    const transcription = await transcriber.processAudio(audioFilePath);
```

Comment:
if we already passed the audioFilePath to the constructor, why do we now need to pass it to this instance method?